script/deploy-core.s.sol

```diff
@@ -10,33 +10,53 @@ import {IERC20, LBPair} from "src/LBPair.sol";
 import {LBQuoter} from "src/LBQuoter.sol";
 
 import {BipsConfig} from "./config/bips-config.sol";
+import {LBPairUpgradeableBeacon} from "src/LBPairUpgradeableBeacon.sol";
+import {Upgrades} from "openzeppelin-foundry-upgrades/Upgrades.sol";
+import { Options } from "openzeppelin-foundry-upgrades/Options.sol";
+
 
 contract CoreDeployer is Script {
     using stdJson for string;
 
-    uint256 private constant FLASHLOAN_FEE = 5e12;
+    uint256 private constant FLASHLOAN_FEE = 0.05e16; //0.05%
+    address deployer;
 
     struct Deployment {
         address factoryV1;
         address factoryV2;
-        address multisig;
-        address routerV1;
+        address owner;
         address routerV2;
         address wNative;
+        address lbPairImplementation;
+        address lbPairUpgradeableBeacon;
+        address quoter;
+        address[] quoteAssets;
     }
 
     string[] chains = ["bob_testnet"];
```

Comment:
we need to make it explicit 
pls replace this with `--chain <CHAIN>` instead - we are unlikely to do multiple upgrades in one go, so we don't need this as an array

Reply:
i dont't think --chain param is available to determine the network, instead we need to use --chain-id int, which i don't think it's user friendly, so i'd prefer to keep use the current impl